h/templates/deform/form.jinja2

```diff
@@ -6,7 +6,7 @@
 {% set form_message_class = 'form-actions-message' %}
 {% endif %}
 <form id="{{ field.formid }}"
-      action="{{ field.action }}"
+      {% if field.action %} action="{{ field.action }}" {% endif %}
```

Comment:
Could you fix the whitespace control here so this doesn't add a line if the action isn't set?